fitbit-react/src/Components/Buttons/PrimaryButton.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+// Here onClick prop is a function which defines what to do on a button click
+// If onCLick is not passed it takes a function which does nothing
+function PrimaryButton({ buttonText, onClick = () => {} }) {
+  return (
+    <button className="button button-primary" onClick={onClick()}>
```

Comment:
not a proper way to pass a function on onClick

Reply:
Done